src/main/resources/reference.conf

```diff
@@ -53,6 +53,12 @@ common {
 
   input = "gs://open-targets-pre-data-releases/21.09.4/input"
   error = ${common.output}"/errors/"
+  additional-outputs = [
+    {
+      format = "json"
+      path = "gs://open-targets-pre-data-releases/21.09.4/output/etl/json"
```

Comment:
Exactly, I think it may be worth looking into it before we introduce more complexity in the logic and / or the configuration file, even if that complexity is not that much.
Should we put the PR on hold until the next release?